scripts/jirautils.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# This script contains jira related utilties for Jenkins.
+
+
+import click
+from jira import JIRA
+
+
+@click.command()
+@click.option('--tag',
+              help='Jenkins build tag',
+              required=True)
+@click.option('--link',
+              help='Link to related build in Jenkins UI',
+              required=True)
+@click.option('--project',
+              help='Jira Project Key',
+              required=True)
+@click.option('--user',
+              help="Jira User",
+              required=True)
+@click.option('--password',
+              help="Jira Password",
+              required=True)
+@click.option('--type',
+              help="Jira issue type",
+              default="Task")
+@click.option('--instance',
+              help="Jira instance url",
+              default="https://rpc-openstack.atlassian.net")
+@click.option('--label',
```

Comment:
I'd insert the line
```python
              'labels'
```
below this one. It makes `create_issue` ever so slightly more readable by making it clearer the object represents a collection. Obviously, this would also require `label` to be changed to `labels` in the function.
http://click.pocoo.org/5/parameters/#parameter-names